tests/test-suite.ts

```diff
@@ -43,10 +44,12 @@ export class TestSuite {
     messageStore?: MessageStore,
     dataStore?: DataStore,
     eventLog?: EventLog,
+    eventStream?: EventStream,
```

Comment:
I am considering renaming this to `runExternalDependenciesTests` or something to that effect.

I was originally thinking to pull the `eventStream` tests into their own `TestSuite` method, but it seems many of them overlap.

Any thoughts @thehenrytsai

Reply:
Went with `runInjectableDependentTests`